src/Atmos/Parameterizations/CloudPhysics/Microphysics.jl

```diff
@@ -1,271 +1,550 @@
 """
     one-moment bulk Microphysics scheme
 
-Microphysics parameterization based on the ideas of Kessler_1995:
+Included processes
   - condensation/evaporation as relaxation to equilibrium
   - autoconversion
   - accretion
-  - rain evaporation
-  - rain terminal velocity
+  - evaporation and sublimation
+  - terminal velocity
 """
 module Microphysics
 
+using SpecialFunctions
+
 using ClimateMachine.MoistThermodynamics
+
 using CLIMAParameters
 using CLIMAParameters.Planet: ρ_cloud_liq, R_v, grav
-using CLIMAParameters.Atmos.Microphysics:
-    C_drag,
-    MP_n_0,
-    K_therm,
-    τ_cond_evap,
-    τ_sub_resub,
-    τ_acnv,
-    q_liq_threshold,
-    E_col,
-    D_vapor,
-    ν_air,
-    N_Sc,
-    a_vent,
-    b_vent
+using CLIMAParameters.Atmos.Microphysics
 
 const APS = AbstractParameterSet
+const ASuspPS = AbstractSuspendedWaterParameterSet
+const AFallPS = AbstractFallingWaterParameterSet
```

Comment:
`FallingWater` -> `Precip`. (The immediate FallingWater association is https://fallingwater.org [which I love])

Reply:
done